### PR TITLE
Add deployment files to .gitignore

### PR DESCRIPTION
**Pull request by tdphillips**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -70,3 +69,10 @@ settings.py
 
 # Documentation
 doc/build
+
+# Deployment
```
Comment (tdphillips) [inline .gitignore]: These are added when we collect static files.

File: .gitignore [hunk 2]
```diff
@@ -3,8 +3,7 @@
 *.db
 *.flv
 *.wmv
-static/*.gz
```
Comment (tdphillips) [inline .gitignore]: These two were woefully inadequate - gz files are being generated more than one level deep in the static directory.

I checked. We have no gz files in the repo currently.

**Comment by koonsies** [comment]
Not sure what is up with the failing check here.

**Comment by tdphillips** [comment]
Jenkins timed out contacting pypi.

**Pull request merged**